src/main/g8/build.sbt

```diff
@@ -3,8 +3,7 @@ val scala212Version = "2.12.2"
 
 lazy val root = (project in file(".")).
   settings(
-    name := "dotty-cross-template",
-    description := "Template sbt project that is cross compiled using Dotty and Scala 2",
```

Comment:
Indeed, but the previous description was incorrect: this is an instance of a template, not a template. I prefer to leave it empty since the user will choose his own description for his project, but if you have a better idea feel free to change it.